<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PumpCalibrateWidget</class>
 <widget class="QWidget" name="PumpCalibrateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0,0">
   <property name="bottomMargin">
    <number>100</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_all_3">
         <property name="text">
          <string>全选1</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_1">
         <property name="text">
          <string>1号泵</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_2">
         <property name="text">
          <string>2号泵</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_3">
         <property name="text">
          <string>3号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_4">
         <property name="text">
          <string>4号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_5">
         <property name="text">
          <string>5号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_6">
         <property name="text">
          <string>6号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_7">
         <property name="text">
          <string>7号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_8">
         <property name="text">
          <string>8号泵</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>校准结果:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnStart">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>开始校准</string>
     </property>
     <property name="btnStyle" stdset="0">
      <string>normalBtn</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
